srcpkgs/wayshot/template

```diff
@@ -0,0 +1,17 @@
+# Template file for 'wayshot'
+pkgname=wayshot
+version=1.1.9
+revision=1
+build_style=cargo
+makedepends="scdoc"
```

Comment:
ahould be hostmakedepends, and you need to install the manpages it generates

Reply:
`wayshot.1.scd` is not available in the releases tar.gz file.

Reply:
but its showing the repo https://github.com/waycrate/wayshot/tree/main/docs

Reply:
ah, looks like it was added after the release